docs/buildkitd.toml.md

```diff
@@ -12,6 +12,13 @@ The file path is `/etc/buildkit/buildkitd.toml` for rootful mode,
 The following is a complete `buildkitd.toml` configuration example.
 Note that some configuration options are only useful in edge cases.
 
+This config file has been updated, some options may not be available on older
+buildkit versions:
+- In v0.17.0, `reservedSpace`/`maxUsedSpace`/`minFreeSpace` properties were
+  added on `worker.oci` and `worker.containerd` as well as to their respective
+  `gcpolicy` properties. `gckeepbytes`/`keepBytes` have been deprecated as a
+  result.
```

Comment:
If you want to keep this info here, I suggest adding a `## Changelog` section or similar.

Reply:
Yah, removed the `gckeepstorage` and `keepBytes` docs - happy to point to older versions of the doc :tada: